### DISCUSSION TITLE
Removing box-shadow from shapes

### DISCUSSION THREAD

**Discussion by Melissa-Kay** [phase1geo/Annotator#72] [Ideas]
This is by far the best annotation tool I've come across for ubuntu, but I'm getting a little annoyed that I can't remove (what I think is) the box-shadow (the white/black 1px border on the shapes). I do a fair bit of documentation for work and my previous screen grabs don't have this since I was using the annotation tool built into mac.

**Comment by phase1geo** [comment]
If you are running the latest master branch version of Annotator (from GitHub). There is a new toggle in the outline toolbar that will turn this 1px border outline off or on.
